standalone/src/models/product-type/product-type/product-type-draft/presets/sample-data-b2b/heavy-earthmoving-and-construction-equipment.spec.ts

```diff
@@ -32,6 +32,7 @@ describe(`with heavyEarthmovingAndConstructionEquipment preset`, () => {
               "nl-NL": "Capaciteit",
               "pt-PT": "Capacidade",
             },
+            "level": "Variant",
```

Comment:
This PR seems to be outdates as sample data presets have been removed on the `main` branch.